packages/web-components/package.json

```diff
@@ -45,7 +45,8 @@
   },
   "scripts": {
     "build:prod": "node ./src/scripts/build-prod.js",
-    "start:prod": "sirv public",
+    "start:prod": "serve public/dist -p 5000",
```

Comment:
Sorry, am not following what this is for. Can you explain?

Reply:
@willmcvay This will serve all bundle asset to localhost:5000, there is a command named `yarn start:dev` the the demo-site package which will:

- Bundle web-components and serve it to localhost:5000
- Bundle demo-site with CDN_URL = localhost:5000
- Run the serverless package
- It's allow us to test the demo-site against local web-components package

I haven't figured out the way to make it "live reload".

Reply:
Okay, so I get it @phmngocnghia thanks. So maybe start:prod is confusing me here - maybe start:dev would make more sense. start:prod was implying we would launch in production. Will leave with you though 👍

Reply:
@willmcvay I will change it to serve:prod as it implies we are serving prod static files, and not for launch